src/python/bcc/__init__.py

```diff
@@ -1262,6 +1288,17 @@ def kprobe_poll(self, timeout = -1):
     def free_bcc_memory(self):
         return lib.bcc_free_memory()
 
+    @staticmethod
+    def add_module(modname):
+      """add_module(modname)
+
+        Add a library or exe to buildsym cache
+      """
+      try:
+        lib.bcc_buildsymcache_add_module(BPF._bsymcache, modname)
+      except Exception, e:
```

Comment:
It should be `except Exception as e`. This caused a few builtbot errors.

Reply:
I had fixed this in the latest commit on my forked branch. Let me check if it has been merged properly or not.